readme.md

```diff
@@ -49,6 +55,13 @@ Default: `false`
 
 Only match an exact string. Useful with `RegExp#test` to check if a string is a URL.
 
+##### strict
+
+Type: `boolean`<br>
+Default: `true`
+
+Force URLs to start with a valid protocol or `www`. If set to `false` it'll match the TLD against a list of valid [TLDs](https://github.com/stephenmathieson/node-tlds).
```

Comment:
> If set to `false` it'll match the TLD against a list of valid [TLDs](https://github.com/stephenmathieson/node-tlds).

Shouldn't it be the other way around? `strict` is, like the name implies, more strict and thus preventing you from choosing the TLD you want.